tests/post_training/pipelines/lm_weight_compression.py

```diff
@@ -256,6 +266,30 @@ def collect_data_from_stdout(self, stdout: str):
         stats.fill(stdout)
         self.run_info.stats_from_output = stats
 
+    def _rename_files(self, folder_path, new_name):
```

Comment:
Please refactor the same way it done here: https://github.com/openvinotoolkit/nncf/pull/3321/files#diff-5d01260f55336ab44f9ae6583b583333bdbd46271c48cf70862bb17c33d5c191R513-R521